AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -379,6 +387,11 @@ - (void)migrateDatabase:(void *)db fromVersion:(NSUInteger)version {
     [MSDBStorage executeNonSelectionQuery:migrationQuery inOpenedDatabase:db];
     [self createPriorityIndex:db];
```

Comment:
Don't we need an index for timestamp by the way? We're going to frequently query based on it?

Reply:
Good topic to discuss. Let's do it separately